.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -6,15 +6,28 @@ labels: 'needs: discussion'
 assignees: ''
 
 ---
+<!-- Thank you for contributing. These HTML commments will not render in the issue, but you can delete them once you've read them if you prefer! -->
 
-Those who maintain this project do so as volunteers and we have no control over their schedule or priorities.
+### Proposed change
+<!-- Use this section to describe the feature you'd like to be added. -->
 
-This means if the feature you are proposing is something you'd like to see added the best way to achieve that is for you to organise the effort required to build it. This could be evangelising for the feature, paying someone, doing it yourself, etc.
 
-Even so it might never get merged because the trade off between additional happy users and maintenance burden is not favourable.
+### Alternative options
+<!-- Use this section to describe alternative options and why you've decided on the proposed feature above. -->
 
-**Describe the solution you'd like**
-A clear and concise description of what you want to happen.
 
-**Describe alternatives you've considered**
-A clear and concise description of any alternative solutions or features you've considered.
+### Who would use this feature?
+<!-- Describe the audience for this feature. This information will affect how work is assigned across the maintenance and contributing team. -->
+
+
+### How much time will adding it take?
```

Comment:
I wouldn't ask this question. Even in my day job where we are a well oiled team of people who have worked together for a while on a small code base we are terrible at estimating efforts required. Estimates seem to come in two categories: spot on (for work that takes "no time") and way off (for work where you discover that actually it is much harder than you thought).

Reply:
Can it be rephrased to effort? The goal is to get the person opening the issue to think about how much labour they're asking for. (Not to actually allocate time for the reasons you say!)

Happy to drop though!

Reply:
(Made a suggestion below)

Reply:
I like the idea of getting people to think about how much work it is. As an example https://github.com/jupyter/repo2docker/pull/637 where I think everyone thought it would be a quick job but turned out to be a bit more work and required more experience :-/

Conclusion: I like the idea, I wonder how we deal with the fact that it is super hard/imprecise. Though maybe that doesn't matter as the whole goal is to get people to think about the topic. In which case we don't actually care about the answer (just don't tell anyone we don't care :-) )

Reply:
Yes - exactly! I don't think the _actual_ answer matters in this case, just that folks have thought about it 😆 (and say something at least within the realms of possibility).